openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Alerts.spec.js

```diff
@@ -0,0 +1,262 @@
+    cy.get('[title="Include"]').should('be.visible').click();
+    cy.get('[title="Include"]').eq(1).click();
+
+    //Select Destination
+    cy.get('button').contains('Add Destination').should('exist').click();
+    cy.get('.ant-select-selection-placeholder')
+      .contains('Select Source')
+      .click({ force: true });
+    cy.wait(1000);
+    cy.get('.ant-select-item-option-content').contains('Email').click();
+    cy.wait(500);
+    //Enter email
+    cy.get(
+      '.ant-form-item-control-input-content > .ant-select > .ant-select-selector > .ant-select-selection-overflow'
+    )
+      .eq(1)
+      .click()
+      .type('testuser@openmetadata.org');
+    //Click save
+    cy.get('[type="submit"]').contains('Save').click();
+    toastNotification('Alerts created successfully.');
+    cy.get('table').should('contain', ALERTS.name);
+  });
+
+  it('Create new alert for Test case data asset', () => {
+    interceptURL('GET', '/api/v1/alerts/*', 'createAlert');
+    //Click on create alert button
+    cy.get('button').contains('Create alert').should('be.visible').click();
+    verifyResponseStatusCode('@createAlert', 200);
+    //Enter alert name
+    cy.get('#name').should('be.visible').type(TEST_CASE.testCaseAlert);
+    //Enter description
+    cy.get('#description')
+      .should('be.visible')
+      .type(TEST_CASE.testCaseDescription);
+    //Click on specific data assets
+    cy.get('[title="All Data Assets"]').should('be.visible').click();
+    cy.get('[title="Specific Data Assets"]').click();
+    cy.get('.ant-select-selection-overflow').should('exist');
+    //Select Test case data asset
+    cy.get('.ant-select-selection-overflow')
+      .should('be.visible')
+      .click()
+      .type('TestCase');
+
+    cy.get('.ant-select-dropdown')
+      .contains('Test Case')
+      .scrollIntoView()
+      .should('be.visible')
+      .click();
+
+    //Select filters
+    cy.get('button').contains('Add Filters').should('exist').click();
+    cy.get('#filteringRules_0_name').invoke('show').click();
+    //Select Test results condition
+
+    cy.get('[title="Test Results"]').should('be.visible').click();
+    cy.get('#name').should('be.visible').click();
+    //Select result
+    cy.get('.ant-select-selection-overflow').eq(1).should('be.visible').click();
+    cy.get('[title="Failed"]').should('be.visible').click();
+    cy.get('#name').should('be.visible').click();
+    //Select include/exclude
+    cy.get('[title="Include"]').should('be.visible').click();
+    cy.get('[title="Include"]').eq(1).click();
+
+    //Select Destination
+    cy.get('button').contains('Add Destination').should('exist').click();
+    cy.get('.ant-select-selection-placeholder')
+      .contains('Select Source')
+      .click({ force: true });
+    cy.wait(1000);
+    cy.get('.ant-select-item-option-content').contains('Email').click();
+    cy.wait(500);
+    //Enter email
+    cy.get('.ant-select-selector > .ant-select-selection-overflow')
```

Comment:
Can you add one more alert with Webhook, MSTeams, Slack as well?

Reply:
Adding these tests